keyboards/cutie_club/wraith/info.json

```diff
@@ -0,0 +1,15 @@
+{
+  "keyboard_name": "wraith",
+  "url": "",
+  "maintainer": "amberstarlight",
+  "width": 16,
+  "height": 6,
+	"layouts": {
+        "LAYOUT_ansi": {
```

Comment:
Your QMK Configurator implementation isn't fully functional here. You have `LAYOUT_ansi` and `LAYOUT_iso` described here, but your only layout macro in `keyboards/cutie_club/wraith/wraith.h` is named `LAYOUT`.

- The JSON trees here need to describe the physical layouts used by the layout macros in `wraith.h`
- The layout macro names given in `info.json` need to be a case-sensitive match to those used in `wraith.h`
- Within each tree, the JSON objects need to be in the same order a `keymap.c` file would use with said layout macro

This is my specialty when it comes to QMK, so if you need help with this just let me know. [Here's a relevant doc](https://docs.qmk.fm/#/reference_configurator_support "Configurator Support - QMK Firmware") that may be helpful here as well.